### PR TITLE
Clean up code in parser

### PR DESCRIPTION
**Pull request by andy-ms**
Small improvements made while browsing.

### CODE REVIEW

File: src/compiler/parser.ts [hunk 1]
```diff
@@ -1309,7 +1309,7 @@ namespace ts {
                 case ParsingContext.ObjectBindingElements:
                     return token() === SyntaxKind.OpenBracketToken || token() === SyntaxKind.DotDotDotToken || isLiteralPropertyName();
                 case ParsingContext.HeritageClauseElement:
-                    // If we see { } then only consume it as an expression if it is followed by , or {
+                    // If we see { } then only consume it as an expression if it is followed by `,` or `{`
```
Comment (aozgaa) [inline src/compiler/parser.ts]: Would you consider the following sentence cleaer?
```
Only consume `{ }` as an expression if it is followed by `,` or `,`.
```

Comment (andy-ms) [inline src/compiler/parser.ts]: Nice catch!

File: src/compiler/parser.ts [hunk 2]
```diff
@@ -5652,6 +5642,20 @@ namespace ts {
             return finishNode(importDeclaration);
         }
 
+        function parseImportEqualsDeclaration(fullStart: number, decorators: NodeArray<Decorator>, modifiers: NodeArray<Modifier>, identifier: ts.Identifier): ImportEqualsDeclaration {
+            // ImportEquals declaration of type:
```
Comment (yuit) [inline src/compiler/parser.ts]: nit: could you move this out as JSDOc

**Comment by yuit** [comment]
Just a small request otherwise lgtm :)

**Pull request merged**